cime/config/e3sm/tests.py

```diff
@@ -176,4 +176,34 @@
             )
         },
 
+    #atmopheric tests for ftypes with 2 builds only
+    #ftype2 is a default and tested in other suites for preqx
+    #ftype1 will be added in ftype1 verification pr
+    #1st: preqx ftype0
+    #2nd: preqx ftype4
+    #3rd: theta-l hy ftype0
+    #4th: theta-l hy ftype2
+    #5th: theta-l hy ftype4
+    #6th: theta-l nh ftype0
+    #7th: theta-l nh ftype2
+    #8th: theta-l nh ftype4
+    "eam_preqx_ftypes" : {
+        "time"     : "01:00:00",
+        "tests"    : (
+                 "ERP.ne4_ne4.FC5AV1C-H01A.cam-preqx_ftype0",
```

Comment:
Both these suites should have "share" : True .

Reply:
(assuming you want them to share the same executables)

Reply:
done, thanks. these lines were not tested since i don't even know how to test them. how would i run a whole suite from command line?

Reply:
./create_test $suite 

"share" is not yet implemented in create_test, so don't worry about re-testing it. It's the same as having --single-exe in the create_test call.

Reply:
$suite without machine_compiler? i won't run these then yet.

Reply:
Yes, create_test has always supported:

```
create_test e3sm_developer
```

for example.